tests/gcp/unit_tests/google_service_account/test_iam_bindings_count.py

```diff
@@ -0,0 +1,23 @@
+"""
+Terraform external provider just handles strings in maps, so tests need to consider this
+"""
+from sys import path, stderr
+
+try:
+    path.insert(1, '../../../test_fixtures/python_validator')
+    from python_validator import python_validator
+except Exception as e:
+    print(e, stderr)
+
+
+"""
+    Checks the data specified in the required metadata
+"""
+
+expected_data = {
+    'test_account_count' : '1',
```

Comment:
Not useful to test the same value. Best practice is extremes and somewhere in the middle. I'd suggest null, 1 and greater than 1. Maybe test a members count as well.